plugins/wallabag.koplugin/main.lua

```diff
@@ -63,6 +63,10 @@ function Wallabag:init()
     self.ui.menu:registerToMainMenu(self)
     self.wb_settings = self.readSettings()
     self.server_url = self.wb_settings.data.wallabag.server_url
+    -- Remove any trailing slashes to avoid failures
+    while self.server_url:sub(-1) == "/" do
+      self.server_url = self.server_url:sub(1, -2);
+    end
```

Comment:
As @hius07 wrote you'd want to collapse and remove any number of / if you did this, but I'd rather do so on entry so if this causes any issues it's relatively easy to diagnose ("the program keeps removing my slashes!") rather than completely hidden.

Reply:
So over here:

https://github.com/koreader/koreader/blob/2df4d4531798f017e0e2edb764f3f492cf65f30a/plugins/wallabag.koplugin/main.lua#L1061